dev-scripts/directory-structure-maker/directory-structure-maker.md

```diff
@@ -0,0 +1,104 @@
+# Preparing Choreo Connect U2 Initial Folder Structure
+
+The shell script in this folder creates Choreo Connect initial folder structure. When
+on-boarding a new version with U2 we can use this script to provide initial Choreo-Connect
+executables. Successful execution of the script will generate below folder structure (version name
+will be changed depending on the version that you are using).
+
+```text
+choreo-connect-1.1.0
+├── ADAPTER
+│   ├── $HOME
+│   │   ├── LICENSE.txt
+│   │   ├── adapter-linux-amd64
+│   │   ├── adapter-linux-arm64
+│   │   ├── check_health.sh
+│   │   ├── conf
+│   │   │   ├── config.toml
+│   │   │   └── log_config.toml
+│   │   └── security
+│   │       ├── keystore
+│   │       │   ├── mg.key
+│   │       │   └── mg.pem
+│   │       └── truststore
+│   │           ├── consul
+│   │           │   ├── consul-agent-ca.pem
+│   │           │   ├── local-dc-client-consul-0-key.pem
+│   │           │   └── local-dc-client-consul-0.pem
+│   │           ├── controlplane.pem
+│   │           ├── global-adapter.pem
```

Comment:
Do we need global adapter pem file to be included?

Reply:
Checked in the pack prepared for the 1.0.0 onboarding. In there also `global-adapter.pem` file is there.

Reply:
Let's remove that